language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -195,15 +197,13 @@ public Sentiment analyzeSentiment(String text) throws IOException {
    * Gets {@link Token}s from the string {@code text}.
    */
   public List<Token> analyzeSyntax(String text) throws IOException {
-    AnnotateTextRequest request =
-        new AnnotateTextRequest()
+    AnalyzeSyntaxRequest request =
```

Comment:
@gguuss The analyzeSentiment is still returning Mag and Polarity. It should return score and mag.